### PR TITLE
[Aqara] Smart plug updated

### PR DESCRIPTION
**Pull request by sbkimaqara**
I separated the code according to the device's firmware version.
Different clusters are reported for version 32 and other versions.
- v32: ElectricalMeasurement, SimpleMetering
- Others; such as v41: AnalogInput

### CODE REVIEW

**Review by greens — CHANGES_REQUESTED** [review]
create a sub-driver of this driver that checks the `APPLICATION_VERSION` field as its `can_handle` function

See this driver (and its sub-driver) for an example: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/tree/main/drivers/SmartThings/zigbee-window-treatment/src/axis

**Comment by sbkimaqara** [comment]
> create a sub-driver of this driver that checks the `APPLICATION_VERSION` field as its `can_handle` function
> 
> See this driver (and its sub-driver) for an example: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/tree/main/drivers/SmartThings/zigbee-window-treatment/src/axis

Are you saying to make the current lua file into two sub-drivers separated by version?

File: drivers/SmartThings/zigbee-switch/src/aqara/aqara_version/init.lua [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      [capabilities.refresh.commands.refresh.NAME] = do_refresh,
+    }
+  },
+  zigbee_handlers = {
+    attr = {
+      [OnOff.ID] = {
+        [OnOff.attributes.OnOff.ID] = on_off_handler
+      },
+      [AnalogInput.ID] = {
+        [AnalogInput.attributes.PresentValue.ID] = present_value_handler
```
Comment (greens) [inline drivers/SmartThings/zigbee-switch/src/aqara/aqara_version/init.lua:66]: Do other devices with `version ~= 32` ever report the `PresentValue` attribute? If not, it's perfectly fine to put this function in the parent driver. If you did that, you wouldn't need to have the `utils` file.

Comment (sbkimaqara) [reply drivers/SmartThings/zigbee-switch/src/aqara/aqara_version/init.lua]: Good advice 👍 
- AnalogInput.attributes.PresentValue handler was moved to the parent
- one line functions were deleted

**Comment by sbkimaqara** [comment]
I updated:
1) configured the minimal interval of the CurrentSummationDelivered attribute of the SimpleMetering cluster to reported every 15 minutes. This will be emitted event same interval in the powerConsumptionReport capability as well as energyMeter capability.
2) improved code readability by deleting not reused functions.

**Review by greens — APPROVED** [review]

**Pull request merged**